### PR TITLE
Added backdrop-filter effects to the header

### PR DESCRIPTION
**Pull request by Sschlenker**
Added the transparency effect similar to the design file using
backdrop-filter.

This commit adds styling to the header to simulate what is mocked-up in the design file for the patcher UI. For the time being there is a temporary background image that I am using to test with,

There is also a red cast that you will see on the image which is coming from the #Content component markup, but I did not decide to make edits to that in case someone else it working on that.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/CUModSquad/cu-patcher-ui/pull/4" target="_blank">ReviewNinja

File: src/sass/_header.scss [hunk 1]
```diff
@@ -4,10 +4,24 @@
...
+body {
+  background: url(../images/test-bg.png) no-repeat;
+  background-size: cover;
+}
+
 #Header {
   flex: 0 1 $header-height - $padding-size * 2;
-  background: rgba(0, 0, 0, 0.25);
   padding: $padding-size;
+  background: linear-gradient(rgba(0,0,0, 0.6) 7%, rgba(0,0,0,0.1) 100%);
```
Comment (Sschlenker) [inline src/sass/_header.scss]: Not going to change this at the moment, but the end of the gradient probably needs to be 0.2 opacity when #Content does not have a background alpha going on.  

If the background for #Content is sticking around then 0.1 might suffice.

File: src/ts/header/Header.tsx [hunk 2]
```diff
@@ -53,4 +53,4 @@ export class Header extends React.Component<HeaderProps, any> {
   }
 };
 
-export default Header;
+export default Header;
```
Comment (codecorsair) [inline src/ts/header/Header.tsx]: missing new line at end of file.

File: src/sass/_header.scss [hunk 3]
```diff
@@ -22,4 +36,4 @@
   display: inline;
   color: $header-text-color;
   cursor: pointer;
-}
+}
```
Comment (codecorsair) [inline src/sass/_header.scss]: missing new line at end of file.

Comment (Sschlenker) [inline src/sass/_header.scss]: !fix

File: src/ts/header/Header.tsx (hunk 2 above)
Comment (Sschlenker) [inline src/ts/header/Header.tsx]: !fix

File: src/sass/_header.scss (hunk 3 above)
Comment (Sschlenker) [inline src/sass/_header.scss]: !fixed

File: src/ts/header/Header.tsx (hunk 2 above)
Comment (Sschlenker) [inline src/ts/header/Header.tsx]: !fixed

**Pull request merged**